helpers/helpers.go

```diff
@@ -278,20 +279,40 @@ func RunCommandInput(in io.Reader, cmdname string, args ...string) error {
 
 // RunCommandSilent runs the given command with args and does not print output
 func RunCommandSilent(cmdname string, args ...string) error {
-	_, err := RunCommandOutput(cmdname, args...)
+	_, err := RunCommandOutput(0, cmdname, args...)
+	return err
+}
+
+// RunCommandSilentTimeout runs the given command with timeout + args and does not print output
+func RunCommandSilentTimeout(timeout int, cmdname string, args ...string) error {
+	_, err := RunCommandOutput(timeout, cmdname, args...)
 	return err
 }
 
 // RunCommandOutput executes the command with arguments and stores its output in
 // memory. If the command succeeds returns that output, if it fails, return err that
 // contains both the out and err streams from the execution.
-func RunCommandOutput(cmdname string, args ...string) (*bytes.Buffer, error) {
-	cmd := exec.Command(cmdname, args...)
+func RunCommandOutput(timeout int, cmdname string, args ...string) (*bytes.Buffer, error) {
```

Comment:
not a huge fan of putting the timeout at the beginning of the args list, but I suppose you need to put it before `cmdname string, args ...string`

Reply:
Yeah, unfortunately by language design I can't put it at the end.

Reply:
What about `cmdname string, timeout int, args ...string)` ? That seems better to me.

Reply:
That's more difficult to pass 'entireCmd...'